wrappers/wrapper.rb

```diff
@@ -275,6 +275,12 @@ def assert_wrong_vehicle_shift_preference_with_heuristic(vrp)
       (vrp.vehicles.map(&:shift_preference).uniq - [:minimize_span] - ['minimize_span']).empty? || vrp.preprocessing_first_solution_strategy.to_a.first != 'periodic'
     end
 
+    def assert_no_activity_with_position(vrp)
```

Comment:
since this `assert` is only for one solver, I suggest you to add it in `vroom.rb` instead of `wrapper.rb`

Reply:
All of our asserts are implemented in one place `wrapper.rb` and used by the needing solvers.

If an assert is used by both solvers, then it becomes a vrp.create.check (except certain cases) because it is about invalidity or disagreement of parameters.

Reply:
yes I already explained it's not a best practice: code needs to be defined as near as possible it is used (when it's possible). Somebody has changed this by the past. At the birth of the `vroom.rb` file, existing `assert_***` methods could be for some of them in this file (and were sometimes named `assert_vroom_***`: this was more common)

Reply:
Why don't we open a ticket and move all the asserts at the same time, this PR uses the existing code design and changing this right here doesn't sound a good idea -- one single assert in `vroom.rb` all others in `wrapper.rb` ?

Both vroom and ortools are based on the wrapper class and `wrapper.rb` implements the `inapplicable_solve?` logic which is the place the actual place where these asserts are called and having the asserts defined in here makes sense because asserts are general statement (for exeample `if the vrp has a service with position or not` is a generic question) and some solvers have certain limitations (in this case vroom has this limitation, so it adds this assert `assert_no_activity_with_position` to its list).

Reply:
An assert describes a property of a vrp. Each solver say if he can manage it or not. It is not a property of VROOM. Nevertheless, as we want to try other solvers. Implementing an assert directly within a solver class will just increase the maintenance.

I don't remember specific asserts with a solver name in it. If so, it was a misinterpretation of the assert feature.

Reply:
open a ticket: we will rollback to the initial way to define assert in solvers class with following rule (as we started):
- if assert is used by only one solver, define it in this solver
- if assert is used by more than one solver, define it in wrapper base class
- if assert is used by all solvers: maybe, it should not be an assert but a Exception instead if vrp is not relevant (exemple: assert_no_pickup_timewindows_after_delivery_timewindows ?)

Reply:
Done https://gitlab.com/mapotempo/optimizer-api/-/issues/821